### DISCUSSION TITLE
New installation issue on LS-GL using installer images

### DISCUSSION THREAD

**Discussion by DanOnDeb** [1000001101000/Debian_on_Buffalo#113] [Q&A]
Hi 1000001101000,

It's me again. I took your advice from our last discussion here and tried a clean fresh install (using an SSD) in my Linkstation Pro (LS-GL) that I can tinker with. Unfortunately, it doesn't work. I did everything as described in your Wiki and downloaded the files from:
https://github.com/1000001101000/Debian_on_Buffalo/tree/master/Buster/installer_images/armel_devices

Unfortunately, the box does not boot into installer mode. As it was a new SSD, I though I may have done something incorrectly with the partitioning so I copied the old HDD over to the new SD (using dd) and tried the installer files above but still get the error light and beeping (6 times red flash). 

When I use the uImage and initrd from the original HDD, it works however. So the disk itself seems to be fine. 

I would really like to test your installer solution as I hope it will increase the box' performance, so I would very much appreciate your help here. Any chance I am doing something wrong?

I very much appreciate you keeping those old pieces of hardware alive. 

Thanks for your help!

73
DanOnDeb

**Comment by 1000001101000** [comment]
hmm, 6 red blinks usually means that if found the boot partition but failed to find/load the boot files. I'm honestly not sure exactly what that indicates in this case. One possibility is that the installer image has grown too large for this device since it's usually right up against the limit....I just don't remember if that results in 6 blinks. 

I'm about to start working on the Debian 11 (Bullseye) installer and it is very likely that the devices that require <7MB initrd or <2mb kernel will no longer be able to use the ssh based installer. 

I've already put together a solution for devices/configurations that don't work with the ssh installer which generates a bootable disk image for the target device:
https://github.com/1000001101000/Debian_on_Buffalo/wiki/Alternate-install-method-via-debootstrap-script
 
My suggestion at this point is to give the alternative method a try, it will probably be the main/only option for the LS-GL moving forward.

Let me know if that helps!
